app/controllers/concerns/course/activity_feeds_concern.rb

```diff
@@ -0,0 +1,17 @@
+module Course::ActivityFeedsConcern
+  extend ActiveSupport::Concern
+
+  # Loads recent activity feeds of a given course.
+  #
+  # @param [Course] course The target course.
+  # @param [Integer] count The count of activity feeds that will be loaded. By default it will
+  #   load all activity feeds.
+  # @return [Array<CourseNotification>] Recent activity feed notifications
+  def load_recent_activity_feeds(course, count = :unset)
```

Comment:
and you shouldn't specify the count here, return a relation and just let the controller call limit()